templates/exporter/service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.exporter.enabled -}}
+kind: Service
+apiVersion: v1
+metadata:
+  name: "{{ template "harbor.exporter" . }}"
+  labels:
+{{ include "harbor.labels" . | indent 4 }}
+    component: exporter
+spec:
+  type: ClusterIP
+  ports:
+    - name: http
+      port: 80
```

Comment:
This is not the default prometheus metrics port

Reply:
whatever, it won't be ever merged anyway